guide/english/bootstrap/jumbotron/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Jumbotron
+---
+A jumbotron indicates a big grey box for calling extra attention to some special content or information.
+
+Use a <div> element with class jumbotron to create a jumbotron
+
+**Code Example:**
+
+`<div class="jumbotron">
```

Comment:
Please use three backticks for multi-line code. Additionally, add html following the opening set of backticks for syntax highlighting, like - ```html.

Please review the [style guide for guide articles](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md) for the right way to create and edit guide articles.